gettsim/tax_transfer.py

```diff
@@ -132,45 +137,47 @@ def tax_transfer(
         },
     )
     in_cols = [
-        "m_wage",
-        "m_self",
-        "m_kapinc",
-        "m_vermiet",
-        "renteneintritt",
-        "m_pensions",
-        "w_hours",
-        "ineducation",
-        "zveranl",
-        "child",
-        "m_childcare",
-        "priv_pens_contr",
-        "handcap_degree",
-        "rvbeit",
-        "avbeit",
-        "pvbeit",
-        "alleinerz",
-        "age",
-        "child_num_tu",
+        "bruttolohn_m",
+        "eink_selbstst_m",
+        "kapital_eink_m",
+        "vermiet_eink_m",
+        "jahr_renteneintr",
+        "ges_rente_m",
+        "arbeitsstunden_w",
+        "in_ausbildung",
+        "gem_veranlagt",
+        "kind",
+        "betreuungskost_m",
+        "prv_rente_beit_m",
+        "behinderungsgrad",
+        "rentenv_beit_m",
+        "arbeitsl_v_beit_m",
+        "pflegev_beit_m",
+        "alleinerziehend",
+        "alter",
+        "anz_kinder_tu",
         "year",
```

Comment:
I think we should go for `jahr`, shouldn't we?

Reply:
Changed everywhere to `jahr`. The python variables are left with `year`. We tackle functions and variables separatly, once we set up the new interfaces!